src/helpers/tx/sendTx/sendEVMTx.ts

```diff
@@ -0,0 +1,27 @@
+import { EVMTx } from "types/evm";
+import { TxResult } from "types/tx";
+import { WalletState } from "contexts/WalletContext";
+import { EIPMethods } from "constants/evm";
+import { getProvider } from "../../evm";
+
+export async function sendEVMTx(
+  wallet: WalletState,
+  tx: EVMTx,
+  attribute?: string //eslint-disable-line
+): Promise<TxResult> {
+  try {
+    const provider = getProvider(wallet.providerId)!;
+    const hash = await provider.request<string>({
+      method: EIPMethods.eth_sendTransaction,
+      params: [tx],
+    });
+
+    return { hash, chainID: wallet.chain.chain_id };
+
+    //TODO:
+    //1. get transaction_receipt by polling
+    //2. get id from logs
+  } catch (err) {
+    return { error: "Error encountered while sending transaction" };
```

Comment:
Should log `err` at least, otherwise actual error info is lost.

Reply:
Don't see a need for this yet, but if we ever need the actual error info in calling components, an alternative approach to error creation/throwing would be to return an object `type Error = { inner: Error; message: string }` where `inner` would contain any "actual" errors that occurred (similar to how [C# Exceptions work](https://learn.microsoft.com/en-us/dotnet/api/system.exception.-ctor?view=net-7.0#system-exception-ctor(system-string-system-exception))).

Reply:
> Don't see a need for this yet, but if we ever need the actual error info in calling components, an alternative approach to error creation/throwing would be to return an object type Error = { inner: Error; message: string } 

This is already done in this PR
```ts
export async function sendEVMTx(
  wallet: WalletState,
  tx: EVMTx,
  attribute?: string //eslint-disable-line
): Promise<TxResult> {
```

where TxResult can be 
```
export type TxResult = TxError | TxSuccess;
```

Reply:
> This is already done in this PR

I don't see any error nesting in this PR. 
And there's no apparent reason for there to be one, it's just an idea I wanted to share.